src/components/SwapWidget/swapWidget.stories.tsx

```diff
@@ -22,4 +22,6 @@ export const Default = TemplateSwapWidget.bind({});
 Default.args = {
   isLimitOrderVisible: false,
   showSettings: true,
+  defaultInputToken: '',
+  defaultOutputToken: '',
```

Comment:
Add one more story that represent default token with some value.
for input token you can use `0x5947bb275c521040051d82396192181b413227a3` and for output token you can use `0x60781c2586d68229fde47564546784ab3faca982`